docker-traceability-plugin/src/main/resources/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction/_api.jelly

```diff
@@ -25,200 +25,227 @@
-        </dd>
+        <dt><code>json</code> (default value)</dt>
+        <dt><code>json-pretty</code></dt>
       </dl>
     </li>
   </ul>
-  Call example: 
+  <h3>Managing data</h3>
+  <p>Docker Deployment <strong>SUBMIT</strong> permission is required for all commands.</p>
+
+  <h4 class="entry-point">$(JENKINS_URL)/docker-traceability/submitContainerStatus</h4>
+  <p>Allows to submit the current container status snapshot with a minimal set of
+  parameters. Outputs of <code>docker inspect $(containerId)</code> can be directly 
+  submitted to Jenkins server using this command.</p>
+  <p>Parameters:</p>
   <ul>
-    <li>
-      From test samples:
-      <i>curl http://localhost:8080/jenkins/docker-traceability/submitReport 
-        --data-urlencode json@src/test/resources/org/jenkinsci/plugins/docker/traceability/samples/submitReport.json</i>
-    </li>
+    <li><dfn><code>inspectData</code></dfn>JSON data being generated by <code>docker inspect $(containerId)</code></li>
+    <li><dfn><code>hostId</code></dfn>Optional ID of the source Docker host. Default value - &quot;unknown&quot;</li>
+    <li><dfn><code>hostName</code></dfn>Optional ID of the source Docker host name. Default value - &quot;unknown&quot;</li>
+    <li><dfn><code>status</code></dfn>Optional status of the container. Default value - &quot;NONE&quot; (means that there is no status change)</li>
+    <li><dfn><code>time</code></dfn>Optional event time, the current server's time will be used by default. The time is specified in <a href="https://en.wikipedia.org/wiki/Unix_time">Unix time</a></li>
+    <li><dfn><code>imageName</code></dfn>Optional name of the image. It can be extracted from &quot;docker images&quot; command output</li>
+    <li><dfn><code>environment</code></dfn>Optional free-style string describing the environment</li>
   </ul>
-  
-  <h4>$(JENKINS_URL)/docker-traceability/deleteContainer</h4>
-  Removes an obsolete container reference from the DockerTraceability registry. 
-  The data from container fingerprints won't be deleted, so the record will be 
-  completely restored after a new submission if its gets deleted due to a mistake.
-  <p/>
-  Parameters:
+  <p>Call examples:</p>
+  <code class="block">curl http://localhost:8080/jenkins/docker-traceability/submitContainerStatus --data-urlencode inspectData="$(docker inspect CONTAINER_ID)"</code>
+
+  <h4 class="entry-point">$(JENKINS_URL)/docker-traceability/submitReport</h4>
+  <p>Submits a report using the extended JSON API. This endpoint can be used by scripts to submit the full available info about the container and its environment in a single command.</p>
+  <p>Parameters:</p>
   <ul>
-    <li><b>id</b> - ID of the container. Only full 64-symbol IDs are supported</li>
+    <li><dfn><code>json</code></dfn>JSON structure being generated by an external script, which includes all the available data regarding the container.</li>
   </ul>
+  <p>This structure has the following entries:</p>
```

Comment:
The description below describes the structure of the <code>json</code> field. The new formatting may be quite confusing, a text should be adjusted at least